toast-tk-runtime/src/main/java/io/toast/tk/runtime/block/ActionSentenceMappingProvider.java

```diff
@@ -1,10 +1,14 @@
 package io.toast.tk.runtime.block;
 
+import java.io.File;
+import java.io.FileInputStream;
 import java.io.IOException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.IOException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)